packages/ui/src/elements/PreviewButton/index.tsx

```diff
@@ -4,25 +4,29 @@ import type { PreviewButtonClientProps } from 'payload'
 import React from 'react'
 
 import { ExternalLinkIcon } from '../../icons/ExternalLink/index.js'
-import { usePreviewURL } from './usePreviewURL.js'
 import './index.scss'
+import { usePreviewURL } from '../../providers/LivePreview/context.js'
+import { useTranslation } from '../../providers/Translation/index.js'
 
 const baseClass = 'preview-btn'
 
 export function PreviewButton(props: PreviewButtonClientProps) {
-  const { generatePreviewURL, label } = usePreviewURL()
+  const { previewURL } = usePreviewURL()
+  const { t } = useTranslation()
+
+  if (!previewURL) {
+    return null
+  }
 
   return (
     <button
```

Comment:
This can now be an anchor tag for better a11y. The user will also be able to copy/paste the href once again ftw.

Reply:
We wouldn't need onClick events is the point here, just my negligence to not change it